apprentice-action/readme.md

```diff
@@ -0,0 +1,7 @@
+# Apprenticeship Interview Action
+This action will execute some endpoint tests (HTTP calls) against your application.
+It will ensure that your application has the expected behaviour.
+
+## How to use
+1. Import the action to your Workflow
```

Comment:
This should probably include example code of how to include the action in a workflow

Reply:
Yeah Micah and Austin are gonna work on the documentation part of this 😄 !